bindings/javascript/__test__/limbo.spec.mjs

```diff
@@ -95,14 +98,63 @@ test("Test pluck(): Rows should only have the values of the first column", async
   db.prepare("CREATE TABLE users (name TEXT, age INTEGER)").run();
   db.prepare("INSERT INTO users (name, age) VALUES (?, ?)").run("Alice", 42);
   db.prepare("INSERT INTO users (name, age) VALUES (?, ?)").run("Bob", 24);
-  let stmt = db.prepare("SELECT * FROM users").pluck();
+  let stmt = db.prepare("SELECT * FROM users").raw().pluck();
```

Comment:
Now it's very unclear what this does and my first thought would be that we are setting both "raw" and "pluck" on, but apparently we are testing the behavior `"setting pluck() after setting raw() sets pluck to false"`. Could this be tested separately?